content/sql/concepts/commands/terms/drop-table/drop-table.md

```diff
@@ -0,0 +1,30 @@
+---
+Title: 'DROP TABLE'
+Description: 'Deletes an existing table in a database.'
+Subjects:
+  - 'Data Science'
+Tags:
+  - 'Database'
+  - 'Queries'
+  - 'PostgreSQL'
+  - 'MySQL'
+  - 'SQLite'
+  - 'Tables'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+The **`DROP TABLE`** command deletes an existing table in a database.
+
+## Syntax
+
+```pseudo
+DROP TABLE table_name;
+```
```

Comment:
Add an explanatory sentence after the ```pseudo block, like, "The above drops the table `table_name` from the database."

Reply:
@SSwiniarski Thank you!  Will do.

Reply:
@SSwiniarski 
I added an explanation after `pseudo` block.
Could you please review it when you have a chance?
Thank you!